src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -15,18 +15,19 @@
 
 namespace Microsoft.AspNetCore.Sockets.Client
 {
-    public class Connection : IConnection
+    public class HttpConnection : IConnection
```

Comment:
yeah but nobody uses namespaces below the `using` declarations, so they don't work that way.